examples/camera-app/linux/src/clusters/camera-avstream-mgmt/camera-av-stream-manager.cpp

```diff
@@ -95,16 +100,30 @@ Protocols::InteractionModel::Status CameraAVStreamManager::VideoStreamModify(con
         }
     }
 
-    ChipLogError(Camera, "Allocated video stream with ID: %d not found", streamID);
+    if (!foundStream)
```

Comment:
Why do we need this foundStream variable?  If we found the stream, we returned already, so if we get here, we did not find it....

Reply:
Yes. Oversight. For Modify and Deallocate we don't need this variable. Removed.